### PR TITLE
Add redshift and downsampling to repository

### PR DESCRIPTION
**Pull request by mariusandra**
Taking over the world, two steps at a time.

### CODE REVIEW

**Review by neilkakkar — APPROVED** [review]
Go for downsampling, waiting for other approvals before merge :)

**Review by Twixes — COMMENTED** [review]
Downsampling and Redshift are OK for me.
Only not sure if this plugin-based approach to tracking sessions is optimal, that smells like something that should be core analytics to me a bit.

**Comment by mariusandra** [comment]
Fair enough, removed for now

**Pull request merged**
